packages/admin/admin-theme/src/componentsTheme/MuiCardHeader.ts

```diff
@@ -0,0 +1,23 @@
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiCardHeader: GetMuiComponentTheme<"MuiCardHeader"> = (component, { spacing, palette }) => ({
```

Comment:
The Buttons are slightly off-center, as you can see in the `active` state. 
Also according to the design the `CardHeader` should have a height of `60px`. Please adjust the paddings accordingly, so it's exactly `60px` with a single line of text. 

<img width="539" alt="Screenshot 2024-12-30 at 12 37 05" src="https://github.com/user-attachments/assets/0091e251-5c27-4a50-9174-3ad63a5e5f71" />

Reply:
Sure, adapted here: 5f20ec260dd622277e6c4272947d3400e0ba4b98

<img width="582" alt="Screenshot 2024-12-30 at 14 13 00" src="https://github.com/user-attachments/assets/3c5922b3-1f78-4662-a5e5-07d9851788c7" />

Reply:
As discussed, I corrected it one more time:

<img width="536" alt="Screenshot 2024-12-30 at 14 51 56" src="https://github.com/user-attachments/assets/7ea7fe15-fdf1-4cac-9b90-30d140022658" />